cmd/statusd/utils.go

```diff
@@ -972,7 +959,7 @@ func testCompleteMultipleQueuedTransactions(t *testing.T) bool {
 
 		time.Sleep(1 * time.Second) // make sure that tx complete signal propagates
```

Comment:
Do we have confidence that this will always work, do we need to find a means to guarantee that signal has propagated like we need to do for node sync?

Reply:
With the sync PR  we would not need to worry about this anymore.